training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    print(f'SageMaker Execution Role:{role}')
</pre><p>Nit - add spaces before all formatted strings for easier readability, ie <code>Role: {role}</code>. 3 occurrences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>